EcoSoapBank/App/Assets.xcassets/ShuttleGrey.colorset/Contents.json

```diff
@@ -0,0 +1,20 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0x74",
+          "green" : "0x6B",
+          "red" : "0x65"
+        }
```

Comment:
Looks like some of your colors are using hex and some are using decimal values. Not a huge deal IMO, but if we wanted to be picky about consistency it might be a good idea to make them all one or the other.

Reply:
I can make sure they are all hex in the assets, didn't realize that switching the UI in there actually changed the resulting json